plenum/test/util/test_common_util.py

```diff
@@ -45,7 +45,7 @@ def test_friendly_exception_formatting_multiple_exceptions():
     ex3 = SigningException()
     ex3.__cause__ = ex2
 
-    expected = '{} [caused by {} [caused by {}]]'.format('SigningException()', ex2.reason, ex1.reason)
+    expected = '{} [caused by {} [caused by {}]]'.format(ex3, ex2, ex1)
```

Comment:
The same here: I think `'SigningException()', ex2.reason, ex1.reason` is more clear

Reply:
Agree, I made the needed changes.